file/genre.json

```diff
@@ -0,0 +1 @@
+[{"type":"Genre","name":"Comedy"},{"type":"Genre","name":"Funk"},{"type":"Genre","name":"\"some genre\""}]
```

Comment:
[OPTIONAL] Nice job adding some sample data to start with, just try to use the same format to keep it clean.